rep-2005.rst

```diff
@@ -0,0 +1,568 @@
+  * `rcl_interfaces/lifecycle_msgs <https://index.ros.org/p/lifecycle_msgs/>`_
+  * `rcl_interfaces/rcl_interfaces <https://index.ros.org/p/rcl_interfaces/>`_
+  * `rcl_interfaces/rosgraph_msgs <https://index.ros.org/p/rosgraph_msgs/>`_
+  * `rcl_interfaces/statistics_msgs <https://index.ros.org/p/statistics_msgs/>`_
+  * `rcl_interfaces/test_msgs <https://index.ros.org/p/test_msgs/>`_
+  * `test_interface_files/test_interface_files <https://index.ros.org/p/test_interface_files/>`_
+  * `unique_identifier_msgs/unique_identifier_msgs <https://index.ros.org/p/unique_identifier_msgs/>`_
+
+* RMW
+
+  * `rmw/rmw <https://index.ros.org/p/rmw/>`_
+  * `rmw/rmw_implementation_cmake <https://index.ros.org/p/rmw_implementation_cmake/>`_
+  * `rmw_implementation/rmw_implementation <https://index.ros.org/p/rmw_implementation/>`_
+  * Connext (Connext itself is not open source)
+
+    * `rmw_connext/rmw_connext_cpp <https://index.ros.org/p/rmw_connext_cpp/>`_
+    * `rmw_connext/rmw_connext_shared_cpp <https://index.ros.org/p/rmw_connext_shared_cpp/>`_
+    * `rosidl_typesupport_connext/connext_cmake_module <https://index.ros.org/p/connext_cmake_module/>`_
+    * `rosidl_typesupport_connext/rosidl_typesupport_connext_c <https://index.ros.org/p/rosidl_typesupport_connext_c/>`_
+    * `rosidl_typesupport_connext/rosidl_typesupport_connext_cpp <https://index.ros.org/p/rosidl_typesupport_connext_cpp/>`_
+
+  * CycloneDDS
+
+    * `cyclonedds/cyclonedds <https://index.ros.org/p/cyclonedds/>`_
+    * `rmw_cyclonedds/cyclonedds_cmake_module <https://index.ros.org/p/cyclonedds_cmake_module/>`_
+    * `rmw_cyclonedds/rmw_cyclonedds_cpp <https://index.ros.org/p/rmw_cyclonedds_cpp/>`_
+
+  * FastRTPS
+
+    * `fastcdr/fastcdr <https://index.ros.org/p/fastcdr/>`_
+    * `fastrtps/fastrtps <https://index.ros.org/p/fastrtps/>`_
+    * `rmw_fastrtps/rmw_fastrtps_cpp <https://index.ros.org/p/rmw_fastrtps_cpp/>`_
+    * `rmw_fastrtps/rmw_fastrtps_dynamic_cpp <https://index.ros.org/p/rmw_fastrtps_dynamic_cpp/>`_
+    * `rmw_fastrtps/rmw_fastrtps_shared_cpp <https://index.ros.org/p/rmw_fastrtps_shared_cpp/>`_
+    * `rosidl_typesupport_fastrtps/fastrtps_cmake_module <https://index.ros.org/p/fastrtps_cmake_module/>`_
+    * `rosidl_typesupport_fastrtps/rosidl_typesupport_fastrtps_c <https://index.ros.org/p/rosidl_typesupport_fastrtps_c/>`_
+    * `rosidl_typesupport_fastrtps/rosidl_typesupport_fastrtps_cpp <https://index.ros.org/p/rosidl_typesupport_fastrtps_cpp/>`_
+
+  * OpenSplice
+
+    * `rmw_opensplice/rmw_opensplice_cpp <https://index.ros.org/p/rmw_opensplice_cpp/>`_
+    * `rosidl_typesupport_opensplice/opensplice_cmake_module <https://index.ros.org/p/opensplice_cmake_module/>`_
+    * `rosidl_typesupport_opensplice/rosidl_typesupport_opensplice_c <https://index.ros.org/p/rosidl_typesupport_opensplice_c/>`_
+    * `rosidl_typesupport_opensplice/rosidl_typesupport_opensplice_cpp <https://index.ros.org/p/rosidl_typesupport_opensplice_cpp/>`_
```

Comment:
As this lists target only Foxy and onwards, can the entire OpenSplice section be removed?

Reply:
Works for me; 34b94a0.